src/main/java/filemanager/directorytracker/ScheduleFileDirectory.java

```diff
@@ -6,36 +6,35 @@
 import java.io.IOException;
 import java.nio.file.*;
 import java.time.LocalDateTime;
-import java.util.concurrent.Executors;
-import java.util.concurrent.ScheduledExecutorService;
-import java.util.concurrent.TimeUnit;
 
 public class ScheduleFileDirectory extends TrackerFileDirectory {
 
     private ConverterFromJsonToXmlService converter;
+    private LocalDateTime timeNow;
 
     @Inject
     public ScheduleFileDirectory(ConverterFromJsonToXmlService converter) {
         this.converter = converter;
+        timeNow = LocalDateTime.now();
```

Comment:
It seems that here you are fixing time for all the future executions. Please check if that is what you really want